website/content/docs/concepts/domain-model/auth-methods.mdx

```diff
@@ -31,6 +31,64 @@ The password auth method has the following additional attributes:
 
 - `min_password_length` - (required) The default is 8.
 
+### OIDC auth method attributes
+
+The OIDC auth method has the following additional attributes:
+
+- `account_claim_maps` (optional list) These are a map from custom
+  claims to the standard claims of sub, name and email. These maps are
+  represented as key=value where the key equals the Provider from-claim and the
+  value equals the Boundary to-claim. For example "oid=sub". May be specified
+  multiple times for different to-claims.
+
+- `allowed_audiences` (optional list) Audiences for which the provider
+  responses will be allowed
+
+- `api_url_prefix` (required) The API prefix to use when generating callback URLs
+  for the provider. Should be set to an address at which the provider can reach
+  back to the controller.
+
+- `callback_url` (output read-only) The URL that should be provided to the IdP
+  for callbacks.
+
+- `claims_scopes` (optional list) The claims scope requested. May be specified
+  multiple times.
+
+- `client_id` (required) The OAuth 2.0 Client Identifier this auth method should
+  use with the provider.
+
+- `client-secret` (required) The corresponding client secret.
+  
+- `client_secret_hmac` (output read-only) The HMAC of the client secret returned
+  by the Boundary controller, which is used for comparison after initial setting
+  of the value.
+
+- `description` - (optional)
```

Comment:
I think we should delete `description` since it is include in the first section as one of the attributes that is common to all auth methods.